README.md

```diff
@@ -13,8 +13,10 @@ Download the PDF here:
 I've built the mindmap with MindNode for the Mac. https://mindnode.com
 
 ## Companion Notebook
-This Mindmap/Cheatsheet has a companion Jupyter Notepad that runs through most of the Data Science steps that can be found at the following link:
+This Mindmap/Cheatsheet has a companion Jupyter Notebook that runs through most of the Data Science steps that can be found at the following link:
```

Comment:
Great Catch. Submit it again without the Jupyter link to the install guide and I'll accept the change.